packages/terra-dropdown-button/src/_DropdownButtonBase.jsx

```diff
@@ -79,10 +79,6 @@ class DropdownButtonBase extends React.Component {
       { 'is-compact': isCompact },
       { disabled: isDisabled },
       customProps.className,
-      /* This needs to match terra-hookshot's react-onclickoutside ignore classname or clicking the caret with
-        the dropdown open will cause the dropdown to close and reopen
-      */
-      'ignore-react-onclickoutside',
```

Comment:
By removing this from _DropdownButtonBase, does this cause a double render the dropdown variant?

Reply:
Had it checked in dropDown variant and it does not cause a double render.